src/ts/components/transaction/Fee.tsx

```diff
@@ -20,7 +34,15 @@ class Fee extends React.Component<IFeeProps, IFeeState> {
     throw new Error(t('apiError'))
   }
 
+  componentDidUpdate (prevProps) {
+    if (this.props.toAddress !== prevProps.toAddress || this.props.address !== prevProps.address) {
+      this.updateBalance()
+      this.props.handleFeeChange(this.state.fee)
```

Comment:
we probably can pass this cb func (`handleFeeChange`) to `updateBalance()`(maybe rename to calculateFee())only when the fee needs to be updated

Reply:
yes It can be passed to `updateBalance()`

Reply:
I think so